test/benchmark/benchmark.go

```diff
@@ -118,6 +126,33 @@ func main() {
 	}
 }
 
+// If there is already a schema present, clear it out
+func clearExistingObjects(c *http.Client, url string) {
+	checkSchemaRequest := createRequest(url+"schema", "GET", nil)
+	checkSchemaResponseCode, body, _, err := performRequest(c, checkSchemaRequest)
+	if err != nil {
+		panic("perform request: %v\n" + err.Error())
+	}
+	if checkSchemaResponseCode != 200 {
+		return
+	}
+
+	var dump models.Schema
+	if err := json.Unmarshal(body, &dump); err != nil {
+		panic("Could not unmarshal read response, error: " + err.Error())
```

Comment:
here also you can use: `panic(errors.Wrap(err, "Could not unmarshal read response"))`